src/pb_env.cc

```diff
@@ -253,6 +254,18 @@ EnvironmentManager::ExtractIfNotExtracted(std::string env_path)
 
   bool env_extracted = false;
   bool re_extraction = false;
+
+  // If the path is not a conda-packed file, then bypass the extraction process
+  struct stat info;
+  if (stat(canonical_env_path, &info) == 0 && S_ISDIR(info.st_mode)) {
```

Comment:
I think we need to return an error if stat returns non-zero code. 

Would it be possible to use boost is directory?
https://www.boost.org/doc/libs/1_79_0/libs/filesystem/doc/reference.html#is_directory

Reply:
The current boost installation we have some linking issues with `libtriton_python.so` and `filesystem.hpp`.  We also use `stat` elsewhere in this folder. Lets keep it as is for now and update everything later? I think `boost::exists` is faster than `stat()`.